bridge-evm/docs/src/contracts/utils/ReceiptUtils.sol/library.ReceiptUtils.md

```
@@ -0,0 +1,107 @@
+### asMini
+
+Convert full receipt to mini receipt
+
+
+```solidity
+function asMini(BridgeTypes.FullReceipt memory receipt)
+    internal
+    pure
+    returns (BridgeTypes.MiniReceipt memory mini);
+```
+**Parameters**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)